apps/nirvlife/src/Library/README.md

```diff
@@ -0,0 +1,3 @@
+# library
+
+anything in this directory is targeted for extraction to a component library and should be developed as such
```

Comment:
also add

- i.e. all imports should be a child to this dir (e.g. and not something out of Components)